DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeIDGeneratorTests.swift

```diff
@@ -11,6 +11,7 @@ import UIKit
 
 class NodeIDGeneratorTests: XCTestCase {
```

Comment:
There are no tests for the new behaviour we added, including the problem we solve: querying the same view by two different node recorders.